### PR TITLE
chore: parameterize Makefile and Dockerfile

### LINKED ISSUE #24: Parameterize Makefile and Dockerfile out to loadtest.env
NOTE:  loadtest.env will eventually be replaced by molotov.json

### PR DESCRIPTION
**Pull request by rpappalax**
r? @m8ttyB 

Closes #24

### CODE REVIEW

**Comment by rpappalax** [comment]
Also you can delete the matt_tmp branch.  this is a copy of that.

File: Dockerfile [hunk 1]
```diff
@@ -1,19 +1,15 @@
...
-RUN apt-get update
-RUN apt-get install -y git
-RUN pip3 install -U pip
-RUN pip3 install virtualenv;
-RUN virtualenv venv -p python3;
-RUN . ./venv/bin/activate
-# RUN venv/bin/pip3 install -r requirements.txt;
-RUN venv/bin/pip3 install git+git://github.com/tarekziade/molotov.git@284e47562e17523b4504047dd05b570938596d58
-# RUN apt-get autoremove -y -qq
-# RUN apt-get clean -y
```
Comment (m8ttyB) [inline Dockerfile]: Since we're not using these, can you delete these commented out commands?

File: requirements.txt [hunk 2]
```diff
@@ -1,4 +1,5 @@
 six
-aiohttp
+molotov==0.4
+# aiohttp
 # This is after the 0.3 release, but before a 0.4 release
-https://github.com/loads/molotov/archive/550fc9e.tar.gz
+# https://github.com/loads/molotov/archive/550fc9e.tar.gz
```
Comment (m8ttyB) [inline requirements.txt]: Delete line #5.

File: Dockerfile (hunk 1 above)
Comment (rpappalax) [inline Dockerfile]: check the other right

File: requirements.txt (hunk 2 above)
Comment (m8ttyB) [inline requirements.txt]: I've been chastened

**Comment by m8ttyB** [comment]
r+

**Pull request merged**